README.md

```diff
@@ -108,75 +135,74 @@ Response (in Body):
   "contentType": "pdf",
   "created": "1970-01-01T00:00:00Z",
   "metadata": {},
-  "href": "/file-upload/envelopes/7989913d-b41f-4455-8baa-e8ed0168960c/files/file-id-1/content"
+  "href": "/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653/files/file-id-1/content"
 }
 ```
 
 #### Download File
-Downloads a file from the envelope
+Downloads a file from an envelope.
 ```
 GET   	/file-upload/envelopes/{envelope-id}/files/{file-id}/content
 ```
 | Responses    | Status    | Description |
 | --------|---------|-------|
 | Ok  | 200   | Download a file.  |
 | Not Found | 404   |  File not found. |
-| Internal Server Error | 500   |  INTERNAL_SERVER_ERROR |
 
-Request (Get): localhost:8898/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653/files/file-id-1/content
+Request (GET): localhost:8898/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653/files/file-id-1/content
 
-Response: Download File
+Response: Downloads a binary file which contains the selected file.
```

Comment:
maybe not downloads but rather "Binary file..." ans response is not a verb but a thing, noun